src/Item/ItemList.java

```diff
@@ -0,0 +1,48 @@
+package Item;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Random;
+import java.util.Set;
+
+public class ItemList {
+  private List<Item> items;
+  Random random = new Random();
+
+  public ItemList() {
+    initializedItems();
+  }
+    // 아이템 리스트에 아이템 객체들 추가
+  public void initializedItems() {
+    // 아이템을 빈 리스트로 초기화
+    items = new ArrayList<>();
+    // 아이템 새로 추가시에 items에 초기 레벨로 추가하는 코드를 추가해주세요.
+    items.add(new AttackSpeedUpItem(0));
+  }
+
+  // 선택화면에 표시하기 위해 무작위로 선택된 아이템 리스트를 얻어오는 메소드
+  public List<Item> getSelectedItemList() {
+    // 상점에 나갈 물품 선정 전에 선행 조건 체크 : 최종 레벨까지 업그레이드를 마친 아이템 제외
+    for (int i = 0; i < items.size(); i++) {
+      if (items.get(i).getLevel() == items.get(i).getMaxLevel()) {
+        items.remove(i);
+      }
+    }
+    Set<Item> tempItemList = new HashSet<>();
+    // 나올 수 있는 아이템의 개수가 3개를 초과하는 경우에만 무작위 선정
+    if (items.size() > 3 ) {
+      // 아이템을 중복을 허용하지 않고 3개를 선택할 때까지 반복(Set이므로 중복은 허용되지 않음)
+      while (tempItemList.size() < 3 && !items.isEmpty()) {
```

Comment:
```suggestion
      while (tempItemList.size() < 3) {
```
이미 if문의 조건으로 items의 사이즈가 3보다 크다는 것을 확인하고 while문이 실행되는 것이기 때문에 비어있는지 확인은 필요 없을 것 같습니다.